terratorch/models/backbones/heliofm_register.py

```diff
@@ -1,9 +1,25 @@
 import logging
+
 import huggingface_hub
-from torch import nn
 import torch
+from terratorch_surya.downstream_examples.ar_segmentation.models import (
```

Comment:
did you test if the code is not breaking in case surya is not present in the venv and you run a different e.g. EO task?